### DISCUSSION TITLE
add photo to album from photo screen

### DISCUSSION THREAD

**Discussion by szamanr** [ente-io/ente#912] [Enhancements]
1. open gallery
2. open a device folder e.g. "whatsapp images"
3. click on a photo to open photo screen

from here i can upload a photo to ente, but i cannot add it to an album.
i can only add a photo to the album from the folder screen by long-pressing a photo and choosing "add to ente". so i would like the same functionality, but from the photo view.

**Comment by ua741** [comment]
By default, we select either the uncategorized section or the album corresponding to the current device folder (if the device folder is marked for backup).

In the photo viewer, we have kept the interactions to a minimum to allow users to swipe through photos and mark the desired ones for backup with a single tap. Users can always organize these later once they have finished marking photos for backup.

We are considering adding a long-press interaction, through which the app will ask you to choose a destination album. What do you think?

**Comment by szamanr** [comment]
seems a bit inconsistent, i would personally prefer to have the option to select the album when pressing the button. but having a long-press interaction or a second button in the menu would also be ok.
